stacker_blueprints/aws_lambda.py

```diff
@@ -219,3 +223,44 @@ def create_template(self):
         self.create_role()
         self.create_function()
         self.create_policy()
+
+
+class FunctionScheduler(Blueprint):
+
+    VARIABLES = {
+        "CloudwatchEventsRule": {
+            "type": TroposphereType(events.Rule),
+            "description": "The troposphere.events.Rule object params.",
+        },
+    }
+
+    def create_scheduler(self):
+        variables = self.get_variables()
+        troposhere_events_rule = variables["CloudwatchEventsRule"]
+        aws_lambda_arns = []
+
+        # append the subject Lambda Function ARN to event rule targets.
+        if "Targets" in troposhere_events_rule.properties:
+            for target in troposhere_events_rule.properties["Targets"]:
+                if target.properties["Arn"].startswith("arn:aws:lambda"):
+                    aws_lambda_arns.append(target.properties["Arn"])
+
+        # schedule a Cloudwatch event rule to invoke our Lambda Function.
+        rule = self.template.add_resource(troposhere_events_rule)
+
+        for aws_lambda_arn in aws_lambda_arns:
+
+            # allow cloudwatch to invoke on any given lambda targets.
+            self.template.add_resource(
+                awslambda.Permission(
+                    "PermissionForCloudwatchEventsToInvokeLambda",
```

Comment:
This is going to have to be unique for every resource, so you can't use a static name.  This is why I mentioned storing the ID above, you can use that in this name.  I'd change it to something like:

`"InvokeLambda%s" % id"`